packages/preview/bob-draw/0.1.0/typst.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "bob-draw"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Luca Ciucci"]
+license = "MIT"
+description = "svgbob for typst, powered by wasm"
+
+homepage = "https://github.com/LucaCiucci/bob-typ"
+repository = "https://github.com/LucaCiucci/bob-typ"
+keywords = [
+    "svg",
+    "ascii",
+    "art",
+    "typst",
+    "wasm",
+    "rust",
+    "svgbob",
+    "bob",
+    "diagram",
+    "graph",
+    "draw",
+]
```

Comment:
could you also add `exclude = ["example.typ", "basic-example.typ", "basic-example.svg"]` (or move the examples into a folder and add `exclude = ["examples"]`)